app/src/main/java/com/pileproject/drive/preferences/BlockPreferencesSchema.java

```diff
@@ -20,15 +20,13 @@
 
 /**
  * Schema for "block_preferences" table
- *
- * @author <a href="mailto:tatsuyaw0c@gmail.com">Tatsuya Iwanari</a>
- * @version 1.0 21-April-2016
  */
 @Table(name = "block_preferences")
 public class BlockPreferencesSchema {
```

Comment:
This class is needed to be moved into main because the thresholds are used in the `selection` blocks. But what is the `default` values for machines. They have different defaults.